sensing/altitude_estimation.hpp

```diff
@@ -74,33 +65,58 @@ typedef struct
 
 
 /**
- * \brief                       Initializes the altitude estimation structure
- *
- * \param   estimator           Pointer to data structure
- * \param   config              Pointer to configuration
- * \param   sonar               Pointer to the sonar
- * \param   barometer           Pointer to the barometer
- * \param   ahrs                Pointer to the ahrs
- * \param   altitude_estimated  Pointer to the estimated altitude
+ * \brief   Default configuration
+ * 
+ * \return  Configuration structure
  */
-void altitude_estimation_init(altitude_estimation_t* estimator,
-                              const altitude_estimation_conf_t* config,
-                              const sonar_t* sonar,
-                              const barometer_t* barometer,
-                              const ahrs_t* ahrs,
-                              altitude_t* altitude_estimated);
+static inline altitude_estimation_conf_t altitude_estimation_default_config(void);
 
 
 /**
- * \brief                   Main update function
- *
- * \param   estimator       Pointer to data structure
+ * \brief   Altitude estimator
  */
-void altitude_estimation_update(altitude_estimation_t* estimator);
+class Altitude_estimation: public Kalman<3,1,1>
```

Comment:
I don't understand the definition of the Kalman, the template states that it should be Kalman<m,n,T> so I do get the m=3, n=1, but I don't get why the type T is equal to 1?